_data/events.yml

```diff
@@ -1236,3 +1236,18 @@
     check the site. Presenter Lukas Zapletal from the Foreman Core team
     can speak Czech or English. These are the languages of choice for the
     event. No hardware should be required, lab with 24 PCs.
+- title: 'Foreman Deep Dives: Foreman-Probing (with Adam Ruzika)'
+  type: 'Livestream'
+  location: virtual
+  start: 2018-03-07 14:00 GMT
+  end: 2018-03-07 15:00 BST
```

Comment:
Is this supposed to be really BST?

Reply:
nope, copy+past problem, I'll fix it, thanks!